web/packages/teleport/src/Discover/SelectResource/SelectResource.tsx

```diff
@@ -380,9 +382,50 @@ export function sortResources(
     return a.name.localeCompare(b.name);
   });
 
+  // Display platform resources first
+  const platform = getPlatform();
+  if (platform.isMac) {
+    setFirstResource(sortedResources, 'macOS');
+  }
+  if (platform.isLinux) {
+    setFirstResource(sortedResources, 'Ubuntu 14.04+');
+    setFirstResource(sortedResources, 'RHEL/CentOS 7+');
+    setFirstResource(sortedResources, 'Debian 8+');
+    setFirstResource(sortedResources, 'Amazon Linux 2/2023');
+  }
+  if (platform.isWin) {
+    setFirstResource(
+      sortedResources,
+      'SQL Server (Preview)',
+      DatabaseLocation.Microsoft
+    );
+    setFirstResource(
+      sortedResources,
+      'SQL Server (Preview)',
+      DatabaseLocation.Azure
+    );
+    setFirstResource(sortedResources, 'Active Directory');
```

Comment:
hrm, i feel hesitant doing it this way... b/c the name could change (in fact some of the linux name will change (consolidate into one with better keywording))

what do you think about, in the file where we hard code this stuff, adding another optional field called idk... `platformDependent`? and `if platformDependent === userPlatform`, set it first in the list?

Reply:
Yep I like that